src/mjolnir/ferry_connections.cc

```diff
@@ -259,6 +261,7 @@ bool ShortFerry(const uint32_t node_index,
     }
   }
   if (short_edge) {
+    UNUSED(wayid);
     LOG_DEBUG("Skip short ferry: way_id = " + std::to_string(wayid));
   }
```

Comment:
```suggestion
#ifdef LOGGING_LEVEL_DEBUG
  if (short_edge) {
    LOG_DEBUG("Skip short ferry: way_id = " + std::to_string(wayid));
  }
#endif
```

Reply:
there's a chain of the variables `wayid(local variable) -> way(local variable) -> ways(method parameter)`. Every one of them is used for that debug log.  So we have to mark at least one of them as `UNUSED` to calm the warning down

Reply:
other than this one suggestion, this pr look good to me!

Reply:
hmm i assume the warning is "assinged to but never read"? to hell with it then lets just remove the log?

Reply:
sure! done